app/views/layouts/application.html.haml

```diff
@@ -8,7 +8,7 @@
     = render partial: 'shared/open_graph'
     = render partial: 'shared/meta'
     %meta{charset: "utf-8"}/
-    %title #{content_for :page_title_prefix} — #{t('app.title')}
+    %title #{content_for :page_title_prefix} — #{t('app.title')} #{"gov.uk"}
```

Comment:
Just wondering how does this look like? There is a `—` between `#{content_for :page_title_prefix} — #{t('app.title')}` but not before `#{"gov.uk"}`

Reply:
It just has an empty space between the two, here's an example: 
![image](https://user-images.githubusercontent.com/47317567/62528314-8de64780-b834-11e9-92f3-6f0ff0a8e5ac.png). Hope that provides more clarity 😄

Reply:
I can  amend it to also add the `-` between Teaching Vacancies and gov.uk if this is desired

Reply:
https://design-system.service.gov.uk/styles/ suggests using `GOV.UK`

Reply:
I dont think we need `-` between Teaching Vacancies and gov.uk